examples/Makefile

```diff
@@ -0,0 +1,53 @@
+.PHONY: ci
+ci: deps checkgofmt vet staticcheck ineffassign predeclared golint errcheck test
+
+.PHONY: deps
+deps:
+	go get -d -v -t ./...
```

Comment:
Something about this upgrades the go.mod automatically, but I'm not sure why.  The one in bigtable (for example) is happy to stay on 1.18 😦

Reply:
Ah ok, I have to roll back testcontainers version to one that supports go 1.18